README.md

```diff
@@ -43,7 +43,7 @@ core.workflow(
         url = "file:///tmp/foo",
     ),
 
-    # Copy everything but don't remove a README_INTERNAL.txt file if it exists.
```

Comment:
Actually it doesn't remove in the destination if exist there :)

Reply:
Oh, I see... :-) I have actually not used copybara, I just stumbled upon the project and that comment seemed weird somehow.

Perhaps it could say

> Copy everything but exclude README_INTERNAL.txt if it exists.

That seems pretty clear to me and is what I would expect from an "exclude" setting.

Reply:
I've updated the branch with this wording -- I hope it's useful to someone :-)